src/Redux/Reactive/StoreExtensions.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Reactive.Linq;
+
+namespace Redux.Reactive
+{
+    public static class StoreExtensions
+    {
+        public static IObservable<T> ObserveState<T>(this IStore<T> store)
+        {
+            return Observable.Create<T>(observer =>
+            {
+                return store.Subscribe(() => observer.OnNext(store.GetState())).Dispose;

```

Comment:
I pulled this down and it won't build here - @GuillaumeSalles does it build for you?